spec/requests/search_results_controller_spec.rb

```diff
@@ -140,4 +140,44 @@
       end
     end
   end
+
+  describe 'GET #show' do
+    context 'given an existing search result ID' do
+      it 'returns a successful response with JSON' do
+        user = Fabricate(:user)
+        Fabricate.times(20, :search_result, user_id: user.id)
```

Comment:
Only create what is needed. Sinc eonly one search result is needed, just create one.